Gems/StartingPointInput/Code/Source/InputConfigurationComponent.cpp

```diff
@@ -22,6 +22,11 @@ namespace StartingPointInput
         return static_cast<AZ::s32>(value);
     };
 
+    InputConfigurationComponent::InputConfigurationComponent()
+    {
+        m_uuid = AZ::Uuid::CreateRandom();
+    }
```

Comment:
The constructor member initialization list syntax should be used here per the [coding standards](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#member-initialization)
```suggestion
    InputConfigurationComponent::InputConfigurationComponent()
        : m_uuid{ AZ::Uuid::CreateRandom() }
    {
    }
```

Reply:
Fixed